docs/Testing-Releases.md

```diff
@@ -21,44 +20,12 @@ While the team remains diligent in keeping API compatibility within FHIR version
 
 You have two options depending on the severity of the issue: 
 1. You may open an [issue](https://github.com/microsoft/fhir-server/issues/new/choose) against the Github repository. We actively triage these and will work on this as best effort. 
-1. If you have an Azure API for FHIR account, you can also raise a support ticket with concern on the next release.
+2. If you have an Azure API for FHIR account, you can also raise a support ticket with concern on the next release.
```

Comment:
markdown tip: you can repeat `1.` in markdown and it will assume a numbered list :)

```
1. a
1. b
```
renders as:
1. a
1. b